strings-csharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringInCSharp/GenerateStreamFromStringMethods.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text;
+
+namespace GenerateStreamFromStringInCSharp
+{
+    public static class GenerateStreamFromStringMethods
+    {
+        public static Stream GenerateStreamFromStringWithStreamWriterAndMemoryStream(string sampleString, Encoding? encoding = null)
+        {
+            encoding ??= Encoding.UTF8;
+
+            var stream = new MemoryStream(encoding.GetByteCount(sampleString));
+            var writer = new StreamWriter(stream, encoding);
+            writer.Write(sampleString);
+            writer.Flush();
+            stream.Position = 0;
+
+            return stream;
+        }
+
+        public static Stream GenerateStreamFromStringWithMemoryStreamAndGetBytes(string sampleString, Encoding? encoding = null)
```

Comment:
As explained in the article review, please break this into multiple lines. Read the guidelines about code formatting if you want a reminder to see how this should be applied properly.